docs/topics/secured-mission-common-interaction.adoc

```diff
@@ -10,7 +10,7 @@ The Java client is packaged as an executable jar and is located at target/sso-cl
 To run the client, use `java -jar target/sso-client.jar -app ...`.
 The full set of command arguments that you can specify can be seen by passing -help:
 +
-[source,options="nowrap",subs="attributes+"]
+[source,bash,options="nowrap",subs="attributes+"]
 ----
 [sstark sso]$ java -jar target/sso-client.jar --help
```

Comment:
@rhoads-zach Maybe we should remove sstark's username from the snippet here. WDYT?